README.md

```diff
@@ -4,7 +4,131 @@
 - [GoDoc](https://godoc.org/go.linecorp.com/centraldogma)
 - [CLI manual](https://line.github.io/centraldogma/client-cli.html)
 
-## How to build
+## Usage
+
+### Getting started
+
+```go
+import "go.linecorp.com/centraldogma"
+
+// create client with OAuth2 token
+centraldogma.NewClientWithToken(baseURL, token, nil)
+```
+
+### Customize transport
+
+If transport is `nil` (like above), `http2.Transport` is used by default. You could inject your own transport easily:
+
+```go
+import "golang.org/x/net/http2"
+
+tr := &http2.Transport{
+    DisableCompression: false,
+}
+
+// create client
+centraldogma.NewClientWithToken(baseURL, token, tr)
+```
+
+### Example
+
+```go
+package sample
+
+import (
+	"context"
+	"sync/atomic"
+	"time"
+
+	"go.linecorp.com/centraldogma"
+)
+
+// CentralDogmaFile represents a file in central dogma.
```

Comment:
How about `a file which contains information of an endpoint in Central Dogma server.`?

Reply:
@minwoox  according to central dogma (https://line.github.io/centraldogma/concepts.html):
`repository is where the configuration files of your application are stored actually`

So how about changing to: `represents a file in application repository, stored on Central Dogma server.`

Reply:
Ah! that's better. Thanks!

Reply:
And this is example. So we should keep something `visually`, and easy to user. While using `contains information of endpoints`, it's a little bit confuse.

In this example, I only want to show user how to manage a single file entry.